committeeoversightapp/models.py

```diff
@@ -272,11 +273,16 @@ class CommitteeRating(models.Model):
     total_hearings = models.IntegerField(null=True, blank=True)
     chp_points = models.IntegerField(null=True, blank=True)
 
+    @property
+    def length_handicap(self):
+        return settings.DEFAULT_CONGRESS_LENGTH / self.congress.length
```

Comment:
The is the opposite of how you described the approach in your PR description?

> This PR adds a length attribute to our Congress model, then uses that length to calculate handicaps for Congresses based on (length / DEFAULT_CONGRESS_LENGTH).

Reply:
Yep this got mixed up, thanks. 

This has shifted since, but: when Congress is inactive for longer than expected, like for the current one, I want the handicap to be > 0. So with the new `inactive_days` attribute, this is what `handicap` looks like. 

```python
@property
def handicap(self):
    '''
    On average, we expect a Congress to be inactive for 62 days of its two year
    session. This accounts for Congresses that are inactive for longer; they
    should have a handicap > 1.
    '''
    extra_inactive_days = self.inactive_days - settings.DEFAULT_CONGRESS_INACTIVE_DAYS
    handicap =  self.length_in_days / (self.length_in_days - extra_inactive_days)

    return handicap
```